tests/IntegrationTests/WorkflowExecutor.IntegrationTests/Features/TaskDestinations.feature

```diff
@@ -90,6 +90,13 @@ Scenario: Multiple task destinations with condition true, multiple task dispatch
     When I publish a Task Update Message Task_Update_Task_Multiple_Destination_Condition_True with status Succeeded
     Then 3 Task Dispatch events are published
 
+@TaskDestinationConditions
+Scenario: Multiple task destinations based on Dicom data condition of not case sensitive, mutiple task dispatch messages sent
```

Comment:
I'm not sure this is the clearest title, could it be something like "Multiple task destinations based on Dicom data with upper and lower cases condition , mutiple task dispatch messages sent"?